services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
```

Comment:
If argument can be `None` you mush show it in a type hint
```suggestion
        genres_ids: list[int] | None = None,
        actors_ids: list[int] | None = None,
```